src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -84,6 +84,11 @@ private static boolean isValidPath(Path filePath) {
      * @throws StorageOperationException if there were errors converting and/or storing data to file.
      */
     public void save(AddressBook addressBook) throws StorageOperationException {
+    	
+    	if (!path.toFile().exists()){
+    		System.out.println("File Not Found!");
```

Comment:
- Good idea to catch the exception in the save()

- But the you way you have handled it is not acceptable. You must set a flag to check it the file does not exist. Then you recover from it and then you should THROW AN EXCEPTION. 

- System.out is not acceptable. 

- Finally you must catch the exception in a proper place. Hint: Main.java executeCommand()